src/app/api/api.component.html

```diff
@@ -24,18 +24,11 @@
         <!--
         <a routerLink="{{ urlHandler.getSPARQLDoc() }}" class="a-displacement">
         -->
-        <a href="{{ urlHandler.getSPARQLDoc() }}" target="blank">
-          <img src="../assets/rdf_w3c_icon.64.png"> &nbsp;
-          SPARQL endpoint
-        </a>
-      </li>
-      <li>
-        <a href="{{ urlHandler.getAPIDoc() }}" target="blank" class="a-displacement">
```

Comment:
@suzialeksander I'm wondering if you meant to delete this line:
```
<a href="{{ urlHandler.getAPIDoc() }}" target="blank" class="a-displacement">
```
I assume it would break the API endpoint link here:
![image](https://github.com/user-attachments/assets/e5134d43-d3d2-4341-a077-c087475fd5f9)
Is this intended?

Reply:
nope, not intended. thanks for catching that